Test/Utils/ContrailAPI/ConfigureDNS.ps1

```diff
@@ -0,0 +1,174 @@
+. $PSScriptRoot\Constants.ps1
+. $PSScriptRoot\Global.ps1
+. $PSScriptRoot\VirtualDNSData.ps1
+. $PSScriptRoot\VirtualDNSRecordData.ps1
+
+function Set-IpamDNSMode {
+    Param (
+        [Parameter(Mandatory = $true)] [string] $ContrailUrl,
+        [Parameter(Mandatory = $true)] [string] $AuthToken,
+        [Parameter(Mandatory = $true)] [string[]] $IpamFQName,
+        [Parameter(Mandatory = $true)]
+            [ValidateSet("none","default-dns-server","tenant-dns-server","virtual-dns-server")]
+            [string] $DNSMode,
+        [Parameter(Mandatory = $false)] [string[]] $TenantServersIPAddresses = @(),
+        [Parameter(Mandatory = $false)] [string] $VirtualServerName
+    )
+
+    $IpamUuid = FQNameToUuid -ContrailUrl $ContrailUrl -AuthToken $AuthToken `
```

Comment:
This variable is not used until much later, maybe let's move it closer to where it's used.